### ISSUE TITLE
Increase the limit of maxinum BPM

### ISSUE THREAD

**Issue by hyeonsu06** [LMMS/lmms#6625]
# Please search the issue tracker for existing feature requests before submitting your own. Delete this line to confirm no similar request has been posted yet.

### Enhancement Summary
Add option of removing BPM limit (example name : "Enable unlimited BPM")

#### Justification
In order to make extratone songs, so I tried to increase the limit of the maxinum BPM which end up with suceess without any bugs. As long as CPU can stand, the limit of BPM can be increased infinitely (Personally almost 50k+ BPM on AMD Ryzen 7 5800H).

#### Mockup
(This example is 1058 BPM)

https://user-images.githubusercontent.com/66936518/215323997-b7bf8698-4fd8-4059-9043-1b9cfb6513fd.mp4

**Comment by Monospace-V** [comment]
I'd prefer decimal bpm over higher bpm limits since any number above 999 can be emulated (varying levels of finesse) with decimals (672.5->1345), but no number between 1000 and 2000 can accurately represent, say, 240.7 bpm.
Unless "higher bpm" pushes to 10,000 in which case it's fine

**Comment by qnebra** [comment]
Well, using pianoroll and 1/192 notes you can effectively go into 12 000 BPM at maximum, or in another word, have note lenght duration of 0,005 second. With snapping to grid at 1/64 which can be done in 1.3, well I think lmms can be absolute insanity for extratone.

**Comment by Monospace-V** [comment]
I agree, you can do a lot with snapping, but what about unusual non-3/2 factor numbers like .7, .1 etc? Can't really do 108.7 bpm as a tempo. Some songs are at strange tempos, and if both options offer >1000 in some form then I'd take the one which allows more precision points.

**Comment by teknopaul** [comment]
Isn't the limit human?

999 BPM is 16.65 beats per second, i.e. 16.65 Hz Its reaching the human audible frequency range already, no?
At which point there is no "beat" its a tone.

50,000 BPM is 833Hz its a fairly high pitched tone.

i.e. a computer can consider anything a beat, but a human has limits.
